lib/roby/test/execution_expectations.rb

```diff
@@ -1145,30 +1164,49 @@ def to_s
             end
 
             class Maintain < Expectation
-                def initialize(at_least_during, block, description, backtrace)
+                def initialize(at_least, block, description, backtrace)
```

Comment:
You're making the difference between at_least and until in the user API (which is good) and then stop making it in this constructor to finally re-split on type... 

Just propagate the same two arguments to the constructor.

Reply:
I didn't want to break the API

Reply:
Please do, these are meant to be private. The public interface are the expectation API calls.

(and add a `# @private` documentation line with the `Maintain` class)